webapp/lib/model.g.dart

```diff
@@ -60,6 +76,22 @@ class Conversation {
     return pubSubClient.publishDocChange(collectionName, docIds, {"tags": newValue?.toList()});
   }
 
+  Future<bool> removeFromTagIds(DocPubSubUpdate pubSubClient, String newValue) {
```

Comment:
same as above - consider renaming to `removeTagId`.

Reply:
Good point. Fixed.